internal/cmdrender/runtime/container.go

```diff
@@ -41,6 +44,8 @@ type ContainerFnPermission struct {
 // ContainerFn implements a KRMFn which run a containerized
 // KRM function
 type ContainerFn struct {
+	Ctx  context.Context
```

Comment:
I think it would make more sense here to pass the context into the `Run` function.

Reply:
Yes, you are right. But in this case `Run` method is part of an interface that exists in `kyaml` library, so had to go with this.